optimisation/ScanSelectionsTree.py

```diff
@@ -108,6 +108,11 @@
 hCrossSecPrompt = inFileCrossSec.Get(inputCfg['predictions']['crosssec']['histonames']['prompt'])
 hCrossSecFD = inFileCrossSec.Get(inputCfg['predictions']['crosssec']['histonames']['feeddown'])
 
+#background correction factor
+if inputCfg['BkgCorrFactor']['filename']:
```

Comment:
Also here same comment than previous ones 😉 And these are the config files: [config_Dplus_pp5TeV_Optimisation.yml](https://github.com/DmesonAnalysers/DmesonAnalysis/blob/master/optimisation/config_Dplus_pp5TeV_Optimisation.yml), [config_Ds_MC_Optimisation_ITS3.yml](https://github.com/DmesonAnalysers/DmesonAnalysis/blob/master/optimisation/config_Ds_MC_Optimisation_ITS3.yml), and [config_Ds_pp5TeV_Optimisation.yml](https://github.com/DmesonAnalysers/DmesonAnalysis/blob/master/optimisation/config_Ds_pp5TeV_Optimisation.yml)